src/config/navigation/defaultStackConfig.tsx

```diff
@@ -444,68 +501,82 @@ export const defaultStackConfig = ({
       })
     },
     {
+      initialParams,
       routeName: ScreenName.VolunteerMe,
       screenComponent: VolunteerMeScreen
     },
     {
+      initialParams,
       routeName: ScreenName.VolunteerPersonal,
       screenComponent: VolunteerPersonalScreen,
       screenOptions: { title: texts.screenTitles.volunteer.personal }
     },
     {
+      initialParams,
       routeName: ScreenName.VolunteerRegistered,
       screenComponent: VolunteerRegisteredScreen,
       screenOptions: { title: texts.screenTitles.volunteer.home }
     },
     {
+      initialParams,
       routeName: ScreenName.VolunteerRegistration,
       screenComponent: VolunteerRegistrationScreen,
       screenOptions: { title: texts.screenTitles.volunteer.home }
     },
     {
+      initialParams,
       routeName: ScreenName.VolunteerSignup,
       screenComponent: VolunteerSignupScreen,
       screenOptions: { title: texts.screenTitles.volunteer.home }
     },
     {
+      initialParams,
       routeName: ScreenName.VoucherDetail,
       screenComponent: VoucherDetailScreen,
       screenOptions: getScreenOptions({ withBookmark: false })
     },
     {
+      initialParams,
       routeName: ScreenName.VoucherHome,
       screenComponent: VoucherHomeScreen,
       screenOptions: { title: texts.screenTitles.voucher.home }
     },
     {
+      initialParams,
       routeName: ScreenName.VoucherIndex,
       screenComponent: VoucherIndexScreen
     },
     {
+      initialParams,
       routeName: ScreenName.VoucherLogin,
       screenComponent: VoucherLoginScreen,
       screenOptions: { title: texts.screenTitles.voucher.home }
     },
     {
+      initialParams,
       routeName: ScreenName.VoucherScanner,
       screenComponent: VoucherScannerScreen,
       screenOptions: { title: texts.screenTitles.voucher.qr }
     },
     {
+      initialParams,
       routeName: ScreenName.WasteCollection,
       screenComponent: WasteCollectionScreen,
       screenOptions: { title: texts.screenTitles.wasteCollection }
     },
     {
+      initialParams,
       routeName: ScreenName.WasteReminder,
       screenComponent: WasteReminderScreen,
       screenOptions: { title: texts.screenTitles.wasteCollection }
     },
     {
+      initialParams,
       routeName: ScreenName.Weather,
       screenComponent: WeatherScreen
     },
     {
+      initialParams,
```

Comment:
why is every route needing the initial params?

Reply:
Because every screen can be tabbed and screens may need param. InitialParams value may not be valid for all screens, but I felt the need to add it in case you need it.